streamer/src/nonblocking/quic.rs

```diff
@@ -501,17 +501,14 @@ async fn setup_connection(
                         stats.clone(),
                     ),
                     |(pubkey, stake, total_stake, max_stake, min_stake)| {
-                        let peer_type = if stake > 0 {
+                        // At least 1 stream out of the all streams allowed in 100 ms window
+                        let min_ratio = 1_f64 / (MAX_STREAMS_PER_MS * 100) as f64;
```

Comment:
Is `100` to align with the throttling interval? Can we use `STREAM_THROTTLING_INTERVAL_MS` instead of this hardcoded value?